data/yam/agama/auto/default_sle.json

```diff
@@ -1,6 +1,7 @@
 {
   "product": {
-    "id": "SLES_16.0"
+    "id": "SLES_16.0",
+    "registrationCode": "{{SCC_REGCODE}}"
```

Comment:
In order to merge we need this change in a different PR to merge it when we really have registration ready.

Reply:
We could create a new _json_ without _scc_ placeholder, just to keep track of the needed format, eg:
- `default_sle.json`
- `default_sle_scc.json`

and run the VRs with `default_sle_scc.json`.

Reply:
scc is by default, so the final final name should not contain scc, you would need to do it in reverse or have a separate PR.